### PR TITLE
cnats 1.4.4 (new formula)

### PR DESCRIPTION
**Pull request by andrelucas**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cnats.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    (testpath/"test.c").write <<-EOS.undent
+      #include <nats.h>
+      #include <stdio.h>
+      int main() {
+        printf("%s\\n", nats_GetVersion());
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.c", "-L#{lib}", "-lnats", "-o", "test"
+    system "./test"
```
Comment (woodruffw) [inline Formula/cnats.rb]: Can we use `shell_output` and `assert_match` to compare this to `version` here?

**Comment by dunn** [comment]
```
Error: 1 problem in 1 formula
cnats:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```
It's close, though, so if another maintainer wants to merge it I won't object.

File: Formula/cnats.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      #include <nats.h>
+      #include <stdio.h>
+      int main() {
+        printf("%s\\n", nats_GetVersion());
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.c", "-L#{lib}", "-lnats", "-o", "test"
+    script_version = shell_output "./test", 0
+    assert_equal "1.4.4", script_version.strip
```
Comment (woodruffw) [inline Formula/cnats.rb]: You should be able to do `assert_match version, script_version` here.

Comment (woodruffw) [inline Formula/cnats.rb]: (`version` gets populated from the `url`)

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**